examples/todo-mvc/ui/widgets/selection_list_view.slint

```diff
@@ -72,11 +73,11 @@ export component SelectionListViewItemDelegate {
                 overflow: elide;
             }
         }     
-
            
         IconButton {
             y: (parent.height - self.height) / 2;
             icon: Icons.remove;
+            accessible-label: @tr("remove");
```

Comment:
It would be far more practical to support the delete key as well. In a proper TODO list app you would use the up and down arrow keys to select listview elements, pressing tab could put the focus to the currently selected item's remove button, but preferably not the other ones.

Reply:
True, full keyboard navigation support would be nice to have. For now I've changed this to "Delete Task".